Compilation error on CentOS 7 of root V6.22.02 c++17

ROOT Version: v6-22-02
Platform: CentOS 7
Compiler: gcc 8.3 c++17

```
[ 80%] Built target GenVector
In file included from input_line_16:3:
/home/annie/Documents/root-6.22.02/build/include/ROOT/RAxis.hxx:57:4: error: exception specification of explicitly defaulted default constructor does not match the calculated one
   RAxisBase() noexcept = default;
   ^
Error: /home/annie/Documents/root-6.22.02/build/bin/rootcling: compilation failure (/home/annie/Documents/root-6.22.02/build/lib/libROOTHist93b0963531_dictUmbrella.h)
make[2]: *** [hist/histv7/CMakeFiles/G__ROOTHist.dir/build.make:108: hist/histv7/G__ROOTHist.cxx] Error 1
make[1]: *** [CMakeFiles/Makefile2:22334: hist/histv7/CMakeFiles/G__ROOTHist.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/hist/histv7/inc/ROOT/RAxis.hxx
+++ b/hist/histv7/inc/ROOT/RAxis.hxx
@@ -54,7 +54,7 @@ protected:
    ///\}
 
    /// Default construct a RAxisBase (for use by derived classes for I/O)
-   RAxisBase() noexcept = default;
+   RAxisBase() noexcept(noexcept(std::string())) = default;
 
    /// Virtual destructor needed in this inheritance-based design
    virtual ~RAxisBase();

